packages/zoe/test/z2spec/escrow-exo.test.js

```diff
@@ -0,0 +1,75 @@
+// @ts-check
+import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
+
+import { makeHeapZone } from '@agoric/zone';
+import { AmountMath, makeDurableIssuerKit } from '@agoric/ertp';
+import { makePromiseKit } from '@endo/promise-kit';
+import { objectMap } from '@endo/patterns';
+import { prepareEscrowExchange } from '../../src/z2spec/escrow-exo.js';
+
+test('escrowExchange in heap zone', async t => {
+  const z1 = makeHeapZone();
+  const makeEscrowExchange = prepareEscrowExchange(z1);
+  const kit = {
+    Money: makeDurableIssuerKit(z1.mapStore('MB'), 'Money'),
+    Stock: makeDurableIssuerKit(z1.mapStore('SB'), 'Stock'),
+  };
+  const { make } = AmountMath;
+  const { Money, Stock } = harden({
+    Money: kit.Money.brand,
+    Stock: kit.Stock.brand,
+  });
+  const issuers = harden({ Money: kit.Money.issuer, Stock: kit.Stock.issuer });
+
+  /**
+   * @template {{ give: AmountKeywordRecord, want: AmountKeywordRecord}} P
+   * @param {string} name
+   * @param {P} proposal
+   */
+  const makeParty = (name, proposal) => {
+    const { give, want } = proposal;
+    const [[giveKW, giveAmt]] = Object.entries(give);
+    const purses = {
+      Money: issuers.Money.makeEmptyPurse(),
+      Stock: issuers.Stock.makeEmptyPurse(),
+    };
+    const purse = purses[giveKW];
+    purse.deposit(kit[giveKW].mint.mintPayment(giveAmt));
+    const payment = purse.withdraw(giveAmt);
```

Comment:
deposit/withdraw gymnastics is left-over from flailing when I couldn't figure out why balances were all 0 after `.run()`
I realized `decisionP` didn't entail depositing payouts; just escrow.